packages/graph/hash_graph/tests/testdata/lib.rs

```diff
@@ -0,0 +1,4 @@
+//! Test data for testing the Graph API.
```

Comment:
This folder should be called `test_data` I think?

Reply:
Shouldn't really matter, but renamed for consistency with the module names: 8cbcb6384776f5c5ce6195e5a23d07dd33ca8fe8